backend/database_handler/validators_registry.py

```diff
@@ -19,7 +19,7 @@ def _parse_validator_data(self, validator_data: dict) -> dict:
             "stake": float(validator_data["stake"]),
             "provider": validator_data["provider"],
             "model": validator_data["model"],
-            "config": json.loads(validator_data["config"]),
+            "config": validator_data["config"],
```

Comment:
@bradleySuira are we sure that this doesn't break other features?

Reply:
this will be the same fix that we apply to staging in another pr (https://github.com/yeagerai/genlayer-simulator/pull/273)[https://github.com/yeagerai/genlayer-simulator/pull/273]